### ISSUE TITLE
Add Trigger Browser covers settings window

### ISSUE THREAD

**Issue by MeestorX** [bitfocus/companion#2040]
### Is this a bug in companion itself or a module?

- [X] I believe this to be a bug in companion

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

When adding a new Trigger (I tested using a Feedback trigger), and using the Browse function, after adding a new trigger, the Edit trigger window pops under the Browse window. The Edit should pop up to the right, or the Browse window should be to the left (or both) so one doesn't cover up the other.
![image](https://user-images.githubusercontent.com/29129658/172057207-be41cfb7-04bc-4ce8-bbf4-293b3a6b663f.png)

### Steps To Reproduce

Add a new trigger and use the Browse button to add it.
Trigger Browser window covers up the Edit window.
Clicking "Done" on the Browser window reveals the Edit Window.
See image, arrow shows where Edit Window has "popped under" the Browser window.

### Expected Behavior

Browser window not covering up the Edit window

### Environment (please complete the following information)

```markdown
- OS: MacOS
- Browser: Safari
- Companion Version: 2.2.0+4447
```

**Comment by Julusian** [comment]
Im going to call this an enhancement rather than a bug, as this is working as intended even if it isnt the best (I didnt want to put effort into rethinking the layout when editing triggers).

I think this is really part of a larger task of reworking various bits of the ui to be more consistent

**Comment by MeestorX** [comment]
Fair enough. If I had more experience with the new Browser, I would have realized that clicking "Done" is the solution. I was clicking around trying to move windows and things and getting nowhere, so I figured other might have the same issue until they realize that you need to click "Done" to move that window out of the way. Especially when you can't see that there's a hidden window behind the browser.

**Comment by MeestorX** [comment]
Do LOVE that browser, though!! :-)

**Comment by Julusian** [comment]
This is fixed in v3.0, the trigger editor is no longer a modal so this no longer happens.
See #2165
